### PR TITLE
Add an improved README

### PR DESCRIPTION
**Pull request by heidar**
Addresses #4

### CODE REVIEW

**Comment by kjdchapman** [comment]
I like this better than previous, but wonder if we need to go into depth on dependencies when it's dockerized. 

Is it useful to someone coming to it new, or can we just get 'em to grab docker and `make serve` to get going?

**Comment by heidar** [comment]
@kjdchapman Why not both? :D Most people prefer to run things outside of Docker for development. I think we should add a section on running it using Docker though, I think that's where your Makefile comes in. :)

File: README.md [hunk 1]
```diff
@@ -2,35 +2,126 @@
 
 # Apply for postgraduate teacher training
 
-**Apply for postgraduate teacher training** service for candidates to apply to intitial teacher training courses.
+**Apply for postgraduate teacher training** is a service for candidates to apply to intitial teacher training courses.
 
-Ruby on Rails application has been set up with:
+## Installing and Running
```
Comment (himal-mandalia) [inline README.md]: Don't capitalise words unless they're proper nouns, so this should be `Installing and running`

File: README.md [hunk 2]
```diff
@@ -2,35 +2,126 @@
...
+
+Do this with Yarn, not npm.
+
+```
+yarn install
+```
+
+### Running
+
+#### Setting up the Database
```
Comment (himal-mandalia) [inline README.md]: `Setting up the database`

File: README.md [hunk 3]
```diff
@@ -2,35 +2,126 @@
...
+You can run all tests and checks with:
 
 ```
-bundle exec govuk-lint-ruby app config db lib spec --format clang
+bundle exec rake
 ```
+
+See below how to run individual tasks.
+
+#### Running Tests
```
Comment (himal-mandalia) [inline README.md]: `Running tests`

File: README.md [hunk 4]
```diff
@@ -2,35 +2,126 @@
...
+bundle exec rspec
+```
+
+Run [Cucumber](https://cucumber.io) tests with:
+
+```
+bundle exec cucumber
+```
+
+#### Running Linters
```
Comment (himal-mandalia) [inline README.md]: `Running linters`

Comment (kjdchapman) [reply README.md]:
```suggestion
#### Running linters
```

**Pull request merged**